class-00/lab-a/starter-code/js/app.js

```diff
@@ -128,7 +127,7 @@ if(localStorage.busmall){
   allProducts = JSON.parse(localStorage.busmall);

```

Comment:
You should see a linter error here. Fix the issue to see if the app runs correctly.